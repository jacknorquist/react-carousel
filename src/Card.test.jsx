import { it, expect } from "vitest";

import { render, fireEvent } from "@testing-library/react";
import Card from "./Card";
import TEST_IMAGES from "./_testCommon.js";



it("renders without crashing", function () {
    render(<Card caption="Hello" src="hello" currNum='5' totalNum='10' />);
});

it("matches snapshot", function () {
    const { container } = render(<Card caption="Hello" src="hello" currNum='5' totalNum='10' />);
    expect(container).toMatchSnapshot();
});