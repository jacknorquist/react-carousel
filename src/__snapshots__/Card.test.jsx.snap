// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`matches snapshot 1`] = `
<div>
  <div
    class="Card"
  >
    <h4
      class="Card-title"
    >
      Hello
    </h4>
    <img
      alt="Hello"
      class="Card-image"
      src="hello"
    />
    <small
      class="Card-small"
    >
      Image 
      5
       of 
      10
      .
    </small>
  </div>
</div>
`;
